robottelo/ui/base.py

```diff
@@ -163,14 +202,9 @@ def delete_entity(self, name, really, name_locator, del_locator,
         # Make sure that element is really removed from UI. It is necessary to
         # verify that fact few times as sometimes 1 second is not enough for
         # element to be actually deleted from DB
+        self.is_entity_delete = True
         for _ in range(3):
```

Comment:
I would suggest to wrap lines 206-214 with `try ... finally` block, to be 100% sure after exiting the `delete_entity()` variable will be set to `False`, so search function will know for sure it's not `delete_entity` called it.
I know it's 99% not an issue, i can barely imagine such case (negative delete test, which tries to delete and then searches for entity to verify it wasn't actually deleted?), so this comment is not a blocker.